SU-SWS/stanford_profile
# D8CORE-1114: Alter Node Edit Form

# READY FOR REVIEW

# Summary
- Move node edit form additional information below paragraphs.

# Needed By (Date)
- EOD Friday

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Clear all caches
3. Edit a stanford_basic page and see right side column now below paragraph fields
4. Edit some values and save the form. Ensure values saved correctly.
5. Create a new stanford_page type and see new layout applied as well

# Affected Projects or Products
- D8CORE-1114

# Associated Issues and/or People
- D8CORE-1114

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
